src/firefly/js/tables/ui/BasicTableView.jsx

```diff
@@ -75,19 +75,30 @@ const BasicTableViewInternal = React.memo((props) => {
     const adjScrollTop = correctScrollTopIfNeeded(totalColWidths, scrollTop, width, height-headerHeight, rowHeight, hlRowIdx, triggeredBy);
     const adjScrollLeft = correctScrollLeftIfNeeded(totalColWidths, scrollLeft, width, triggeredBy);
 
+    const isSingleColumnTable = (columns) => columns.reduce(
+        (numVisibleCols, col) => get(col, ['visibility'], 'show') === 'show'
+            ? numVisibleCols + 1 : numVisibleCols, 0)
+        === 1;
```

Comment:
I prefer `filter().length` over `reduce` but, you can decide.  Or, you can use `TableUtil.getColsByType(columns)`.  It abstracts the knowledge of 'visibility', 'show', etc.

Reply:
Yeah, filter().length is neater - it didn't occur to me earlier.